format/src/main/scala/org/scalafmt/internal/AssociatedTrivias.scala

```diff
@@ -23,7 +23,11 @@ final case class AssociatedTrivias(
   def trailings(token: Token): Option[Tokens] =
     allTrailings.get(token)
 
-  private def toDoc(tokens: Option[Seq[Token]], isLeading: Boolean): Doc = {
+  private def toDoc(
+      tokens: Option[Seq[Token]],
+      isLeading: Boolean,
```

Comment:
It would be good to have a docstring explaining isLeading and isSeparator since it's not clear what they do